src/main/java/com/metamx/emitter/core/HttpPostEmitter.java

```diff
@@ -445,6 +492,7 @@ private void tryEmitOneFailedBuffer()
     private void tryEmitAndDrainAllFailedBuffers()
     {
       for (FailedBuffer failedBuffer; (failedBuffer = failedBuffers.poll()) != null; ) {
+        approximateFailedBuffersCount.decrementAndGet();
         sendWithRetries(failedBuffer.buffer, failedBuffer.length, failedBuffer.eventCount);
```

Comment:
Another one that counts _before_ the operation.

Reply:
Changed in https://github.com/metamx/java-util/pull/64